serde/src/private/ser.rs

```diff
@@ -19,7 +19,7 @@ pub fn serialize_tagged_newtype<S, T>(
     type_ident: &'static str,
     variant_ident: &'static str,
     tag: &'static str,
-    variant_name: &'static str,
+    variant_name: &'static VariantName,
```

Comment:
It seems that you can remove reference
```suggestion
    variant_name: VariantName,
```